cluster-autoscaler/version/version.go

```diff
@@ -17,4 +17,4 @@ limitations under the License.
 package version
 
 // ClusterAutoscalerVersion contains version of CA.
-const ClusterAutoscalerVersion = "1.19.0-beta.1"
```

Comment:
This should not be changed in a cherry-pick.

Reply:
Hmm, this wasn't me. Should I re-do this PR?

Reply:
Just remove this part of the change.